Docs/introduction/binary-whitelisting-overview.md

```diff
@@ -0,0 +1,31 @@
+# Binary Whitelisting Overview
+
+#### Background
+
+The decision flow starts in the kernel. The macOS kernel is extendable by way of a kernel extension (KEXT). macOS makes available kernel programming interfaces (KPIs) to be used by a KEXT. Santa makes use of the Kernel Authorization (Kauth) KPI. This is a very powerful and verbose interface that gives Santa the ability to listen in on most vnode and file systems operations and to take actions, directly or indirectly, on the operations being performed. Still, there are some limitations to Kauth which are pointed out in the santa-driver document. For more information on the santa-driver KEXT see the santa-driver document.
+
+#### Flow of an exec()
+
+This is a high level overview of the binary whitelisting/blacklisting decision process end to end. For a more a more detailed account of each part, see the respective documents. This flow does not cover the logging component to Santa, see the log document for this information.
```

Comment:
This is a high level overview of the binary whitelisting/blacklisting decision process. For a more a more detailed account of each part, see their respective documentation. This flow does not cover the logging component of Santa, see the log documentation for more info.

link log doc.